### ISSUE TITLE
[Issue]: Episode's Prefered download language doesn't followed parent settings.

### ISSUE THREAD

**Issue by HMSRenown** [jellyfin/jellyfin#12015]
### Please describe your bug

It seems that after version 10.9.0, when I specify the Preferred download language of a specific season of the episode to be different from the default option and refresh the metadata, only the metadata content of this season is refreshed, The season's episodes' metadata will not be refreshed synchronously.
Under Version 10.8.13, This issue does not exist .

### Reproduction Steps

1.In Jellyfin Version 10.9.4
2.Set the default Preferred download language setting of the TV Series Collection to language A, in this case, Chinese Simplified (with Some Metadata fallback to English)
3.Set the Preferred download language for a certain season in the series to Language B, in this case, Russian
4.Refresh Metadata with Replace all Metadata

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS:Windows 10
 - Linux Kernel: none
 - Virtualization: KVM
 - Clients: Browser
 - Browser: Firefox 126
 - ffmpeg version 6.0.1-Jellyfin
 - Playback: Direct Play, Transcode
 - Hardware Acceleration: NVENC
 - GPU Model: NVIDIA RTX 4060Ti
 - Installed Plugins: TMDB, TVDB, OMDB, WEBh
 - Reverse Proxy: Nginx
 - Base URL: 
 - Networking: 
 - Storage:
```

### Jellyfin logs

```shell
Seems no related log to this issue.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fernvenue** [comment]
Can be reproduced.

**Comment by felix920506** [comment]
fixed by #12028
